frontend/src/components/print/print-nuxt/DownloadNuxtPdfListItem.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-list-item @click="generatePdf">
```

Comment:
```suggestion
  <v-list-item @click="generatePdf" :disabled="loading">
```

Reply:
Perfect, here I think it makes sense to disable it completely